bin/Powheg/run_pwg_condor.py

```diff
@@ -366,7 +366,7 @@ def runEvents(parstage, folderName, EOSfolder, njobs, powInputName, jobtag, proc
           f.write('tar xzf WW_MiNNLO_2loop_grids_reduced1.tar.gz\n')
           f.write('ls\n')
         f.write('cp -p ' + rootfolder + '/' + folderName + '/powheg.input.' + parstage + ' ./powheg.input' + '\n') # copy input file for this stage explicitly, needed by condor dag
-        f.write('echo ' + str (i) + ' | ./pwhg_main \n')
+        f.write('echo ' + str (i+1) + ' | ./pwhg_main \n')
```

Comment:
what is the purpose of this modification ?

Reply:
See the last part of my initial comment:

> In addition to this, an additional minor fix in run_pwg_condor.py is included which fixes the job index passed to Powheg during the pre-sampling in stages 2 and 3. For Powheg, indices start at 1 but so far, the first job had index 0, which caused this particular job to crash and thus one independent pre-sampling file less to be produced.

It's not really related to the issue of vRES not reading the pre-sampling, but I stumbled upon it while checking the logs of each job during stages 1-3, so I thought I might as well fix it. :)

Reply:
Excellent!